src/renderer/state.ts

```diff
@@ -539,7 +540,6 @@ export class AppState {
       this.outputBuffer += strData;
       strData = this.outputBuffer;
       const parts = strData.split('\r\n');
-
```

Comment:
It seemed like this extra space violated TSLint rules? But let me know if that's wrong!